Tools/jit/_targets.py

```diff
@@ -212,13 +212,17 @@ def build(
         ):
             return
         stencil_groups = asyncio.run(self._build_stencils())
-        with jit_stencils.open("w") as file:
-            file.write(digest)
-            if comment:
-                file.write(f"// {comment}\n\n")
-            file.write("")
```

Comment:
Not entirely sure what was the purpose of this liíne, but it is no longer there.